added loanCalc app + test using Junit assert

DenisProkhorov opened the pull request:
вычистил репо от старых файлов
Написал приложение и тест, как было сказано в задании + использовал Assert  из JUnit в тесте

Commit:
added loanCalc app + test using Junit assert

Commit:
removed target from track

vkhizhin reviewed (commented):
В принципе, все хорошо. Только есть несколько моментов по-мелочи:
1) В pom.xml не совсем понимаю зачем ты 2 раза указал dependency с junit-ом, причем во 2-ом указал scope - compile (хотя compile и так дефолтный). Достаточно один раз указать dependency со scope - test (т.к. эта либа нужна только для тестов, при компиляции основного кода она нам не нужна). Кстати, если интересно, про скоупы в мавене можно почитать вот тут - https://www.baeldung.com/maven-dependency-scopes
2) В LoanCalcService в методе `createRequest` у тебя переменная `localVar` никак не используется, так что лучше ее убрать
3) В LoanCalcRepository в методе `save` лучше сразу вернуть результат (`return ++requestId`), а не сохранять его в промежуточную переменную `localVar`, т.к. эта переменная нигде больше не используется по факту и никакого полезного действия от нее нет.

Commit:
заапплаил комменты.
Случилось недопонимание - думал, что надо всё сделать, как было на занятии (а там, чтобы показать как работает дебаггер, как раз была локальная переменная и return с созданием новой переменной).
